src/hotspot/cpu/x86/templateInterpreterGenerator_x86_64.cpp

```diff
@@ -191,7 +191,7 @@ address TemplateInterpreterGenerator::generate_CRC32_update_entry() {
     // c_rarg1: scratch (rsi on non-Win64, rdx on Win64)
 
     Label slow_path;
-    __ safepoint_poll(slow_path, r15_thread, rscratch1);
+    __ safepoint_poll(slow_path, r15_thread, true /* at_return */, false /* in_nmethod */);
```

Comment:
Why is this tagged with 'true /* at_return */? Same for line 240. The _x86_32 version uses false.

Reply:
Good point.